composer.lock

```diff
@@ -3487,50 +3487,6 @@
             ],
             "time": "2018-01-29T19:49:41+00:00"
         },
-        {
```

Comment:
Was this meant to be committed?

Reply:
Nope, a quick update and I didn't notice this file snuck in.